templates/nexus-default.properties

```diff
@@ -0,0 +1,10 @@
+# Jetty section
```

Comment:
Ok. Sorry I did not notice how you enable properties override.
I removed nexus-default.properties template and use lineinfile instead.

Is it ok for you ?